pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -256,8 +256,7 @@ func (m *Multicluster) AddMemberCluster(clusterID cluster.ID, rc *secretcontroll
 		}
 		// Patch validation webhook cert
 		if m.caBundleWatcher != nil {
-			controller.NewValidatingWebhookController(client, m.revision,
-				m.secretNamespace, m.caBundleWatcher).Run(clusterStopCh)
+			controller.NewValidatingWebhookController(client, m.revision, m.secretNamespace, m.caBundleWatcher).Run(clusterStopCh)
```

Comment:
Change it to `go controller.NewValidatingWebhookController(client, m.revision, m.secretNamespace, m.caBundleWatcher).Run(clusterStopCh)` And no need to change others, which seems not very right

BTW: donot we have integration tests?

Reply:
We do not have integration tests for this. I changed to Run ValidatingWebhookController in a separate go routine.

I removed requeuing the event.,

Any other changes you mean? I prefer to leave the event in the queue because we can figure out the event type easily without doing bunch of checks

Reply:
Make sense now

Reply:
Unbelievable this code path is not run in multi cluster tests

Reply:
We don't cover "istiod-less" remotes in tests currently. The "remote cluster" type we have documented still runs istiod and runs it's own webhook cert patcher.

Reply:
is it easy to add that test? it would be good to have one.